xun/init/cookiecutter-workflow/{{cookiecutter.project_name}}/__main__.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module file is the entrypoint in case the project package is run with
+`python -m {{cookiecutter.project_name}}`.
+For more info, read:
+    - https://docs.python.org/3/using/cmdline.html#cmdoption-m
+"""
+
+from .{{cookiecutter.project_name}} import capitalize
+import argparse
+import logging
+import xun
+
+
+logging.basicConfig(level=logging.WARNING)
+logging.getLogger('xun').setLevel(logging.DEBUG)
+
+
+"""
+Driver and store setup
+For driver, choose between sequential or dask
+For store, choose between disk, memory, redis or sftp
```

Comment:
I think we should leave out suggestions, as they can change. They should check other docs to see the options